src/_posts/databases/mysql/2000-01-01-connecting.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Connecting to Your Scalingo for MySQL® Addon
+nav: Connecting
+modified_at: 2024-03-04 12:00:00
+tags: databases mysql addon
+index: 3
+---
+
+When a Scalingo for MySQL® addon is provisioned, the platform also adds
+two [environment variables]({% post_url platform/app/2000-01-01-environment %})
+to your app:
+
+- `SCALINGO_MYSQL_URL`: a string containing all information needed to
+  establish a connection between your application and your database. This
+  string is called a **connection URI**.
+- `DATABASE_URL`: an alias for `SCALINGO_MYSQL_URL`.
+
+
+## Understanding the Connection URI
+
+The connection URI is made of several **components** separated one from each
+other by a **delimiter**. For example, the `@` character is used to mark the
+end of the *userinfo* (credentials) component of the URI.
+
+In the case of MySQL®, the connection URI provided by Scalingo is always
+formed as follows:
+
+```bash
+mysql://[user]:[password]@[url]:[port]/[dbname]?useSSL=true&verifyServerCertificate=false
+```
+
+For more information about the connection URI syntax, please see
+[RFC 3986](https://datatracker.ietf.org/doc/html/rfc3986) which defines the URI
+Generic Syntax.
+
+
+## Using the Connection URI
+
+**We strongly advise to use one of the two environment variables available to
+connect your application to your database**. Please don't use the value, but
+the environment variable itself.
+
+While the value of the provided connection URI **should not** change over time,
+**we don't guarantee it**. For example, the URI could change for maintenance
+reasons. In such a case, using the environment variable guarantees that your
+application can restart successfully without human intervention. Otherwise, you
+would have to update your code with the new value and trigger a new deployment,
+which would most probably contribute to a greater downtime.
+
+In most cases, you can pass the environment variable directly to the client
+library you are using. But sometimes, the library requires a specific URI
+format, invididual keypairs or another format. In such cases, you
+need to parse the connection URI to retrieve the required values and build
+what's required by the library you are using. Our advice to use the environment
+variable still remains applicable.
```

Comment:
suggestion: maybe add a sentence to explain that splitting the environment variable must be done in the application code? I have the feeling it's not explicit enough

Reply:
Done!

Reply:
Done in https://github.com/Scalingo/documentation/pull/2501/commits/97fc3f4220030362c83e79981e39350d5ec98433